src/main.py

```diff
@@ -56,7 +58,11 @@ def setup(self):
             self.arena,
         )
 
-        game_mode = StockGameMode(self.player_manager.players)
+        if GAME_MODE == "stock":
+            game_mode = StockGameMode(self.player_manager.players)
+        elif GAME_MODE == "empty":
```

Comment:
Is there a better name for empty?  It's not obvious to me what empty game mode means

Reply:
it was the name that either Andrew or I said in one of our brainstorming sessions. Empty because it had no specific rules of it's own was my understanding. Could be changed to Debug mode or something else potentially